src/common/Touchable.js

```diff
@@ -22,6 +24,7 @@ type Props = {
  * Component to encapsulate our custom and platform-specific
  * settings applied to the built-in touchable components.
  *
+ * @prop [handlePressWithDelay] - Handle press with delay or not.
```

Comment:
This line of documentation really doesn't add anything to the name and type. :-) When that's what we have to say, it's better to just leave it off.

Also, this line should be at the end; the others, like `children` and `onPress`, are much more central to what the component does and should be shown to the reader first.